Readme.md

```diff
@@ -20,6 +20,7 @@ application:
     $ git clone https://github.com/ForNeVeR/kaiwa.git
     $ cd kaiwa
     $ npm install
+    $ tsd install
```

Comment:
I don't like that. I don't have the tsd installed globally and I don't want our users to have it. Will wrap in npm script.